src/CodeGenerator/src/Generator/OperationGenerator.php

```diff
@@ -100,21 +101,21 @@ public function generate(Operation $operation): void
         }
 
         if (null !== $operation->getOutput()) {
-            $resutClass = $this->resultGenerator->generate($operation);
```

Comment:
good catch

Reply:
We dont need to `git blame` this. I know it was me who did this typo. I make typos everywhere =)

Reply:
sure about that? ;-) https://github.com/async-aws/aws/commit/2c3a63a7dccead9b4fe6921227006dab210b159c#diff-a69fae827839ebd310686b9067fc4518R97

Reply:
oh =)